src/JsonSchema/Enum.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace JsonSchema;
+
+class Enum extends \MabeEnum\Enum
+{
```

Comment:
More for internal benefit, I guess. If someone else decides to use Enum for something they can just extend `/JsonSchema/Enum` instead of `/Mabe/Enum`, and if we ever decide to switch to some other Enum implementation we only have to change the code in one place. Just a little future-proofing.